.rubocop.yml

```diff
@@ -3,6 +3,8 @@ AllCops:
   Exclude:
     - "Guardfile"
     - "Rakefile"
+    - "main.rb"
+    - "app.rb"
     - "node_modules/**/*"
```

Comment:
Kindly remove the app and main from excluded linter check

Reply:
Hello @jeanbuhendwa thanks for your great feedback.